lib/node.js

```diff
@@ -25,64 +25,70 @@ declare class Buffer {
+  readUIntLE(offset: number, byteLength: number, noAssert?: boolean): number;
+  slice(start?: number, end?: number): Buffer;
+  swap16(): Buffer;
+  swap32(): Buffer;
+  toJSON(): Array<number>;
+  toString(encoding?: buffer$Encoding, start?: number, end?: number): string;
+  values(): Iterator<number>;
+  write(string: string, offset?: number, length?: number, encoding?: buffer$Encoding): void;
+  writeDoubleBE(value: number, offset: number, noAssert?: boolean): number;
+  writeDoubleLE(value: number, offset: number, noAssert?: boolean): number;
+  writeFloatBE(value: number, offset: number, noAssert?: boolean): number;
+  writeFloatLE(value: number, offset: number, noAssert?: boolean): number;
   writeInt16BE(value: number, offset: number, noAssert?: boolean): number;
-  writeInt32LE(value: number, offset: number, noAssert?: boolean): number;
+  writeInt16LE(value: number, offset: number, noAssert?: boolean): number;
   writeInt32BE(value: number, offset: number, noAssert?: boolean): number;
-  writeIntLE(value: number, offset: number, byteLength: number, noAssert?: boolean): number;
+  writeInt32LE(value: number, offset: number, noAssert?: boolean): number;
+  writeInt8(value: number, offset: number, noAssert?: boolean): number;
   writeIntBE(value: number, offset: number, byteLength: number, noAssert?: boolean): number;
-  writeFloatLE(value: number, offset: number, noAssert?: boolean): number;
-  writeFloatBE(value: number, offset: number, noAssert?: boolean): number;
-  writeDoubleLE(value: number, offset: number, noAssert?: boolean): number;
-  writeDoubleBE(value: number, offset: number, noAssert?: boolean): number;
+  writeIntLE(value: number, offset: number, byteLength: number, noAssert?: boolean): number;
+  writeUInt16BE(value: number, offset: number, noAssert?: boolean): number;
+  writeUInt16LE(value: number, offset: number, noAssert?: boolean): number;
+  writeUInt32BE(value: number, offset: number, noAssert?: boolean): number;
+  writeUInt32LE(value: number, offset: number, noAssert?: boolean): number;
+  writeUInt8(value: number, offset: number, noAssert?: boolean): number;
+  writeUIntBE(value: number, offset: number, byteLength: number, noAssert?: boolean): number;
+  writeUIntLE(value: number, offset: number, byteLength: number, noAssert?: boolean): number;
 
   static alloc(size: number, fill?: string | number, encoding?: buffer$Encoding): Buffer;
-  static isEncoding(encoding: string): boolean;
-  static isBuffer(obj: any): boolean;
+  static allocUnsafe(size: number): Buffer;
+  static allocUnsafeSlow(size: number): Buffer;
   static byteLength(string: string, encoding?: buffer$Encoding): number;
-  static concat(list: Array<Buffer>, totalLength?: number): Buffer;
   static compare(buf1: Buffer, buf2: Buffer): number;
+  static concat(list: Array<Buffer>, totalLength?: number): Buffer;
+  static from(value: Array<number> | number | string | Buffer, encoding?: buffer$Encoding): Buffer;
```

Comment:
i'm tweaking this a little bit while I import it:

- `number` isn't allowed: https://github.com/nodejs/node/blob/0b9e135c3eaf9b83671b1ecbaa1aa4ba700a8add/lib/buffer.js#L130

- `encoding` is only allowed when `value` is a string (except for the `ArrayBuffer` case which we don't implement yet AFAICT)